data/metainfo/org.gnome.Shell.Extensions.GSConnect.metainfo.xml.in

```diff
@@ -58,6 +58,9 @@ SPDX-License-Identifier: CC0-1.0
   </description>
 
   <releases>
+    <release version="58" date="2024-10-09">
```

Comment:
This release date was chosen arbitrarily; consider it tentative as there's no reason to believe it'll be accurate. If it is **NOT**, it should be corrected immediately prior to merge.

Reply:
I don't think that just changing the release version and date needs approval.

Reply:
@Saqr-Sayed 

Hmm. I guess there are two ways I can respond to that:

1. _Technically_, it does need approval — all PRs need approval, because that's how the repo is configured. The branch protection rules won't enable merging a PR until it has one approving review from a committer other than the author.

    As I've said, I have the **ability** to override that rule, but wouldn't except as a last resort. Requiring a second pair of eyes is a smart policy and a sensible level of protection for a public, collaboratively-developed project. If we can't scrounge up one uninvolved reviewer for PRs, then that's a much bigger problem than unmerged PRs and we should probably focus on addressing **that** first.

2. At the moment, this PR ISN'T waiting for approval. (I mean, it is, but even if it had one it wouldn't get merged yet.) At the moment this is waiting for #1866 to be merged. It doesn't make _any_ sense to put together a new release without first getting that change in. Until then, this one doesn't need to go anywhere.